quadrigacx.gemspec

```diff
@@ -27,5 +27,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'vcr',         '~> 2.9'
   spec.add_development_dependency 'gem-release', '~> 0.7'
   spec.add_development_dependency 'bundler',     '~> 1.7'
+  spec.add_development_dependency 'rubocop',     '~> 0.50.0'
```

Comment:
Just noting that this is not the latest version, as we want to support ruby 2.0.